lib/i18n/backend/file_key_value.rb

```diff
@@ -0,0 +1,75 @@
+require 'pathname'
+require 'digest/sha2'
+
+module I18n
+  module Backend
+    # I18n backend for a key-value store that automatically synchronizes
+    # its contents with Ruby or YAML files on the I18n load path.
+    # This backend improves startup time and reduces memory usage,
+    # and allows you to continue using existing i18n files unchanged.
+    class FileKeyValue < KeyValue
```

Comment:
This could use some tests

Reply:
agreed- i'll add some coverage before merging.